FirstProject/assets/new-evaluation.less

```diff
@@ -154,14 +170,175 @@
   width: 1200px;
   margin: 0 auto;
   margin-top: 50px;
+}
 
-  .col-md-3 {
-    display: inline-block;
-  }
+.col-md-3 {
+  display: inline-block;
 }
 
 a.border_bottom:hover {
-  border-bottom: 4px solid blue;
+  border-bottom: 4px solid @blue;
   border-bottom-width: 7px;
 }
 
+
+
+
+
+.language_form {
+  margin-top: 70px;
+}
+
+.description_section {
+  margin-left: 300px;
+}
+
+fieldset.field_section {
+  text-align: center;
+}
+
+legend {
+  text-align: left;
+  color: #03A9F4;
+}
+
+.input-circle[type='radio']{
+  width: 35px;
+  height: 23px;
+  border-radius: 15px;
+  top: -2px;
+  left: -1px;
+  position: relative;
+  background-color: #d1d3d1;
+  content: '';
+  display: inline-block;
+  visibility: visible;
+  border: 2px solid white;
+}
+
+.footer_area {
+  padding: 25px 350px 20px 0;
+  font-weight:bold;
+  font-size:16px;
+  color: gray;
+  text-align: right;
+  margin-top: 120px;
+  margin-bottom: 0;
+}
+
+
+
+
+.language_form {
+  margin-top: 70px;
+}
+
+.description_section {
+  margin-left: 300px;
+}
+
+fieldset.field_section {
+  text-align: center;
+}
+
+legend {
+  text-align: left;
+  color: #03A9F4;
+}
+
+.input-circle[type='radio']{
+  width: 35px;
+  height: 23px;
+  border-radius: 15px;
+  top: -2px;
+  left: -1px;
+  position: relative;
+  background-color: #d1d3d1;
+  content: '';
+  display: inline-block;
+  visibility: visible;
+  border: 2px solid white;
```

Comment:
L252 was a `border-radius` rule, if I'm not mistaken. To me, it makes sense to have them close to one another. :)